src/bin/tikv-ctl.rs

```diff
@@ -57,6 +57,7 @@ fn perror_and_exit<E: Error>(prefix: &str, e: E) -> ! {
 fn new_debug_executor(
     db: Option<&str>,
     raft_db: Option<&str>,
+    skip_paranoid_checks: bool,
```

Comment:
oh, If we will another config later, do we need to add a new param here again? 

why not using this in config directly?

Reply:
You mean add it in `src/config.rs`? I think only `tikv-ctl` needs it, `tikv-server` should never use the flag. So I prefer to leave it in `debug.rs`.